handeye_calibration.json

```diff
@@ -1,24 +1,24 @@
 {
   "calibration_type": "eye_on_base",
   "robot_base_frame": "base_link",
-  "robot_effector_frame": "TCP",
+  "robot_effector_frame": "gripper_tcp",
```

Comment:
@td-code I hope this is fine. Checked the Pickles and they don't seem to have any reference to the `TCP` itself.

The new frame (`gripper_tcp`) has an identical rotation compared to the old `TCP`.